src/envinfo.js

```diff
@@ -83,9 +83,20 @@ function main(props, options) {
   });
 }
 
+function runMainFromConfigFile(configFile, options) {
+  return Promise.resolve(configFile()).then(({ options: configOptions, ...config }) => {
+    return main(config, { ...options, ...configOptions });
```

Comment:
if you switch the order of options and configOptions, I think we can use command line flags to override whats in the config file, I think this is ideal

Reply:
I think we should also be able to merge command line switches for --system as well.

Reply:
@tabrindle I switched the order of `options` and `configOptions` which makes a lot of sense.  

For merging command line switches, shouldn't we be able to merge all command line switches (`--browsers`, `IDEs`, etc)?